<launch>
  <!-- Launch the turtlesim node -->
  <node name="turtlesim_node" pkg="turtlesim" type="turtlesim_node"/>

  <rosparam command="load" file="$(find turtlesim_draw_shapes)/config/square_params.yaml" />
  <node name="square_turtle" pkg="turtlesim_draw_shapes" type="draw_square" />
</launch>
